mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -235,9 +235,36 @@ LogicalResult ConvConverter<ConvType>::matchAndRewrite(
   newShape.push_back(outShape[1]);
   Type newOutTy = RankedTensorType::get(newShape, outputTy.getElementType());
 
+  auto expandTo2D = [&rewriter, &loc](mlir::Value value) {
```

Comment:
migraphx.convolution becomes tosa.conv2d or tosa.conv3d.  There is no tosa.conv1d, so here we expand those into conv2d by adding an extra x1 dimension to the tensors and some dummy values to the attributes.